app/Http/Controllers/Admin/DiagramController.php

```diff
@@ -94,27 +104,49 @@ public function generateDiagram(Request $request, $idWorkspace)
         }
         $locations = $workspace->locations;
         $colorLocation = [];
+
         foreach ($locations as $key => $location) {
             foreach ($location->seats as $id => $seat) {
+                $user_id = $this->userRepository->get();
+                $list_user_id = unserialize($seat->user_id);
                 $colorLocation[$key][$id]['location'] = $location->name;
                 $colorLocation[$key][$id]['seat_id'] = $seat->id;
-                if ($seat->user != null) {
-                    $colorLocation[$key][$id]['user_name'] = $seat->user->name;
-                    $colorLocation[$key][$id]['avatar'] = $seat->user->avatar;
-                    $colorLocation[$key][$id]['user_id'] =  $seat->user->id;
+                if ($seat->user_id != null) {
+                    $checkName = [];
+                    $checkAvatar = [];
+                    $checkUserId = [];
+                    foreach ($user_id as $value) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 118</br>Variable "user_id" is not in valid camel caps format